components/build-service/README.md

```diff
@@ -47,24 +47,10 @@ MY_QUAY_USER=mkovarik PIPELINESASCODE=1 ./hack/build/build-via-appstudio.sh http
 ### Change of default pipeline bundle
 
 Pipeline bundles are generated by [build-definitions](https://github.com/redhat-appstudio/build-definitions).
-By default the bundle is defined in `build-templates` namespace:
-```
-oc get configmap -n build-templates -o jsonpath='{ .data.default_build_bundle }' build-pipelines-defaults
-```
-
-It can be overridden by configmap in working namespace:
-```
-oc create configmap build-pipelines-defaults --from-literal default_build_bundle=$BUNDLE
-```
-
-## HACBS enablement
-
-HACBS workflow can be set by creation of configmap `hacbs` in the user namespace.
 
-`oc create configmap hacbs` will:
+By default the bundle is defined in `components/build-service/build-pipeline-selector.yaml` namespace.
```

Comment:
You refer to a `build-pipeline-selector` object here, but the sentence ends with word `namespace`.